src/main/java/ChatRequest.java

```diff
@@ -0,0 +1,27 @@
+import lombok.*;
+
+import java.util.List;
+
+    /**
```

Comment:
:information_source: The author tag is not necessary and can be considered redundant as version control systems like Git already track authorship. (Severity: Minor)